src/internal.c

```diff
@@ -29485,6 +29493,7 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
                 /* Copy the haveExtendedMasterSecret property from the ticket to
                  * the saved session, so the property may be checked later. */
                 ssl->session.haveEMS = it.haveEMS;
+                ato32((const byte*)&it.timestamp, &ssl->session.bornOn);
```

Comment:
Why was this line added to populate it.timestamp? Should it also be wrapped in `WOLFSSL_NO_TICKET_EXPIRE`?

Reply:
This is because it.timestamp is used to store the ticket creation time set in CreateTicket. The time will be referenced for a later ticket timeout check. We do not need to wrap this line with WOLFSSL_NO_TICKET_EXPIRE macro as it does not hurt to put the ticket creation time in the session ticket.